data/battle_scripts_1.s

```diff
@@ -414,6 +414,61 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectExtremeEvoboost         @ EFFECT_EXTREME_EVOBOOST
 	.4byte BattleScript_EffectTerrainHit              @ EFFECT_DAMAGE_SET_TERRAIN
 
+BattleScript_AffectionBasedEndurance::
+	playanimation BS_TARGET, B_ANIM_AFFECTION_HANGED_ON
```

Comment:
Why not just call `B_ANIM_HANGED_ON` instead of creating a whole new general animation?

Reply:
So people can go to `General_AffectionHangedOn` and customize the animation if they want to :eyes: